src/lib/AccountStoreIndexeddb.js

```diff
@@ -25,7 +24,7 @@ class AccountStore {
     constructor(dbName = 'accounts') {
         this._dbName = dbName;
         /** @type {IDBDatabase | undefined} */
-        this._db;
+        this._db; // eslint-disable-line no-unused-expressions
```

Comment:
It's a ugly to always have to use this linter command. First we add this var only for TS typing, then the TS annotation for the type... and now also this linter command? That's not linter, that's uglier. :P